doc/meetings/build-wg-meeting-2023-07-04.md

```diff
@@ -0,0 +1,81 @@
+# Node.js  Build WorkGroup Meeting 2023-07-04
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=LxUP44zAawM
+* **GitHub Issue**: https://github.com/nodejs/build/issues/3405
+* **Minutes Google Doc**: https://docs.google.com/document/d/1l0rtHesVkymetXOXYitQJFDnFN6UPYNh-uehiSCgg2E/edit
+
+## Present
+
+* Build team: @nodejs/build
+* Ulises Gascon: @ulisesGascon
+* Richard Lau: @richardlau
+* Michaël Zasso: @targos
+* Stewart Addison: @sxa
+
+## Agenda
+
+## Announcements
+
+* Extracted from **build-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/build
+
+* DNS Wildcard problem?[#3404](https://github.com/nodejs/build/issues/3404)
+  * Wildcard was implemented in the past and we don’t have the full context 
+  * This topic was added to the agenda for awareness
+  * @Richard is planning to remove the wildcard and add explicit DNS records, this should not be a breaking change (Cloudflare level change not Nginx)
```

Comment:
We should remove the `@` here (and everywhere else in these notes) unless we want to switch these mentions to use the GitHub handles.
```suggestion
  * Richard is planning to remove the wildcard and add explicit DNS records, this should not be a breaking change (Cloudflare level change not Nginx)
```

Reply:
Great catch!

Reply:
There are many others

Reply:
I did a find/replace in 1f03163 👍